.github/workflows/publish_api.yml

```diff
@@ -8,7 +8,7 @@ env:
 
 jobs:
   build-and-push-image:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux, xlarge]
```

Comment:
this change will require setting up proxy so that images can be pushed to `ghcr`. Unless self hosted runners have been updated and don't require this anymore?

Reply:
Indeed no longer needed. :partying_face:

Reply:
... however, both here and in the frontend image publishing workflow it is necessary to still configure a mirror for docker.io such as not to hit rate limits. Done that in a follow-up commit now.